package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-google-maps",
-  "version": "2.0.20",
```

Comment:
I cut releases it is not done in a PR. A release is done via `gunt bump-@` and you need to have permissions to push the tag. So this will not be accepted.